Formula/earthly.rb

```diff
@@ -29,10 +29,8 @@ def install
     tags = "dfrunmount dfrunsecurity dfsecrets dfssh dfrunnetwork"
     system "go", "build", "-tags", tags, *std_go_args(ldflags: ldflags), "./cmd/earthly"
 
-    bash_output = Utils.safe_popen_read("#{bin}/earthly", "bootstrap", "--source", "bash")
-    (bash_completion/"earthly").write bash_output
-    zsh_output = Utils.safe_popen_read("#{bin}/earthly", "bootstrap", "--source", "zsh")
-    (zsh_completion/"_earthly").write zsh_output
+    generate_completions_from_executable(bin/"earthly", "bootstrap", shell_parameter_format: "--source=",
+                                                                     shells:                 [:bash, :zsh])
```

Comment:
What about this instead?

```suggestion
    generate_completions_from_executable(bin/"earthly", "bootstrap", "--source", shells: [:bash, :zsh])
```